### ISSUE TITLE
3D draw API plan

### ISSUE THREAD

**Issue by junminacts** [kisvegabor/lvgl#7032]
### Problem to solve

LVGL have any plan to add 3D draw API ? Small 3D GPU (with the similar features of OpenGL ES 2.0) will be integrated to IoT devices soon.

### Success criteria

Support 3D draw API to draw 3D model, glTF descripted models (https://www.khronos.org/gltf/), etc.

### Solution outline

can draw 3D models

### Rabbit holes

No

### Testing

draw a 3D box or glTF models 

### Teaching

Not yet

**Comment by kisvegabor** [comment]
I'm not familiar enough with 3D to comment in detail, but I also agree that 3D on MCUs will be important in the future. 

`glTF` sounds great! We already have a built-in driver to create OpenGL windows and add textures there. If I understand correctly we need a `glTF` renderer (similar to a PNG decoder) which can render to an OpenGL or other texture.

**Comment by junminacts** [comment]
Yes, we need a glTF renderer. Beside the glTF, 3d draw API, like vector API declared in lv_draw_vector.h, should be added.
The 3D GPU will be powered by Verisilicon and be similar to OpenGL ES 2.0.

**Comment by kisvegabor** [comment]
It's not forgotten. In the background I'm having and still organizing meetings with some related parties.
